resources/lib/WonderfulSubsBrowser.py

```diff
@@ -97,14 +99,14 @@ def _handle_watchlist_paging(self, results, base_url, page):
         name = "Next Page (%d/%d)" % (next_page, total_pages)
         return [utils.allocate_item(name, base_url % next_page, True, None)]
 
-    def _json_request(self, url, data):
-        response = json.loads(self._get_request(url, data))
-        if response["status"] != 200:
-            raise Exception("Request %s returned with error code %d" % (url,
-                                                                        response["status"]))
+    def _json_request(self, url, data, reauth=True):
+        response = self._get_request(url, data)
 
-        return response["json"]
+        if self._response_forbidden(response) and reauth:
+            login()
+            return self._json_request(url, data, reauth=False)
```

Comment:
Dont you need to just tell xbmc to referesh?
Also, i think there was another place which tried to handle relogin..

Reply:
My knowledge about xmbc is not sufficient for that :)

Reply:
Ill need to referesh my memory but i think there is a function to halt render and just referesh the same addon url

Reply:
II was exploring today, on another issue, and came across: Kodi Documentation website [ https://codedocs.xyz/AlwinEsch/kodi/index.html ].  It appears to be for v19.0, so I do not know if you can find something useful here.  nalch, I thought this might help.  

Also I
 tried your version yesterday and watched 22/7 sub [from Currently Airing].  I did not get any subs, on screen.  I reviewed the log, and found WS could not find sub.  I apologize, but I did not copy the segment of the log referencing this issue...  Sorry, just thought I would mention this.

I really appreciate all you are doing!